recipes/graphene/all/conanfile.py

```diff
@@ -51,7 +52,8 @@ def layout(self):
 
     def requirements(self):
         if self.options.with_glib:
-            self.requires("glib/2.75.2")
+            # leave run=can_run(self) on requires for compatibility reasons.
```

Comment:
what does it mean? compatibility with what?

Reply:
conan 1.x.